Emitron/Emitron/UI/Library/Filtering/FiltersView.swift

```diff
@@ -106,8 +106,7 @@ struct FiltersView: View {
   }
   
   private func applyOrCloseButton() -> MainButtonView {
-    let equalSets = Set(libraryRepository.nonPaginationParameters) == Set(filters.appliedParameters)
-    let title = equalSets ? "Close" : "Apply"
+    let title = "Apply filters"
```

Comment:
Nit - filters should be capitalised

```suggestion
    let title = "Apply Filters"
```